pkg/sqlite/deprecate_test.go

```diff
@@ -0,0 +1,330 @@
+					Object:      []string{"{\"apiVersion\":\"operators.coreos.com/v1alpha1\",\"kind\":\"ClusterServiceVersion\",\"metadata\":{\"creationTimestamp\":null,\"name\":\"1.0.2\"},\"spec\":{\"apiservicedefinitions\":{},\"cleanup\":{\"enabled\":false},\"customresourcedefinitions\":{},\"displayName\":\"\",\"install\":{\"spec\":{\"deployments\":null},\"strategy\":\"\"},\"provider\":{},\"replaces\":\"1.0.1\",\"skips\":[\"1.0.2-rc0\"],\"version\":\"1.0.2\"},\"status\":{\"cleanup\":{}}}"},
+					BundlePath:  "bundle-1.0.2",
+					Version:     "1.0.2",
+				},
+				{
+					CsvName:     "1.0.2-rc1",
+					PackageName: "testpkg",
+					ChannelName: "1.x",
+					CsvJson:     "{\"apiVersion\":\"operators.coreos.com/v1alpha1\",\"kind\":\"ClusterServiceVersion\",\"metadata\":{\"creationTimestamp\":null,\"name\":\"1.0.2-rc1\"},\"spec\":{\"apiservicedefinitions\":{},\"cleanup\":{\"enabled\":false},\"customresourcedefinitions\":{},\"displayName\":\"\",\"install\":{\"spec\":{\"deployments\":null},\"strategy\":\"\"},\"provider\":{},\"replaces\":\"1.0.1\",\"skips\":[\"1.0.2-rc0\"],\"version\":\"1.0.2-rc1\"},\"status\":{\"cleanup\":{}}}",
+					Object:      []string{"{\"apiVersion\":\"operators.coreos.com/v1alpha1\",\"kind\":\"ClusterServiceVersion\",\"metadata\":{\"creationTimestamp\":null,\"name\":\"1.0.2-rc1\"},\"spec\":{\"apiservicedefinitions\":{},\"cleanup\":{\"enabled\":false},\"customresourcedefinitions\":{},\"displayName\":\"\",\"install\":{\"spec\":{\"deployments\":null},\"strategy\":\"\"},\"provider\":{},\"replaces\":\"1.0.1\",\"skips\":[\"1.0.2-rc0\"],\"version\":\"1.0.2-rc1\"},\"status\":{\"cleanup\":{}}}"},
+					BundlePath:  "bundle-1.0.2-rc1",
+					Version:     "1.0.2-rc1",
+					Properties: []*api.Property{{
+						Type:  registry.DeprecatedType,
+						Value: "{}",
+					}},
+				},
+			},
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.description, func(t *testing.T) {
+			logrus.SetLevel(logrus.DebugLevel)
+			db, cleanup := CreateTestDb(t)
+			defer cleanup()
+			store, err := NewSQLLiteLoader(db)
+			require.NoError(t, err)
+			err = store.Migrate(context.TODO())
+			require.NoError(t, err)
+
+			graphLoader, err := NewSQLGraphLoaderFromDB(db)
+			require.NoError(t, err)
+
+			querier := NewSQLLiteQuerierFromDb(db)
+
+			bundleLoader := registry.BundleGraphLoader{}
+			for _, b := range tt.initial {
+				graph, err := graphLoader.Generate(b.pkg)
+				if err != nil {
+					require.EqualError(t, err, registry.ErrPackageNotInDatabase.Error(), "Expected empty error or package not in database")
+				}
+				bndl := newRegistryBundle(t, b)
+				graph, err = bundleLoader.AddBundleToGraph(bndl, graph, &registry.AnnotationsFile{Annotations: *bndl.Annotations}, false)
```

Comment:
Missing error check?
```suggestion
				graph, err = bundleLoader.AddBundleToGraph(bndl, graph, &registry.AnnotationsFile{Annotations: *bndl.Annotations}, false)
				require.NoError(t, err)
```

Reply:
Yeah, agreed we should catch this error as well as the store.AddBundleSemver.